src/api/index.js

```diff
@@ -71,6 +71,15 @@ const userAboutInfo = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+
+const serverDownErrorMessage = (error) => {
+  debugger
+  if (error === 'Failed to fetch') {
```

Comment:
Can we change the error message from backend? if yes then no need for frontend changes. 

Message should be: "Something went wrong try again later"

Reply:
Changed